src/java/org/apache/nutch/scoring/webgraph/LinkDumper.java

```diff
@@ -214,94 +212,98 @@ public void write(DataOutput out) throws IOException {
    * Inverts outlinks from the WebGraph to inlinks and attaches node
    * information.
    */
-  public static class Inverter implements
-      Mapper<Text, Writable, Text, ObjectWritable>,
-      Reducer<Text, ObjectWritable, Text, LinkNode> {
+  public static class Inverter {
 
-    private JobConf conf;
-
-    public void configure(JobConf conf) {
-      this.conf = conf;
-    }
+    private static Configuration conf;
```

Comment:
See comments in Generator etc. regarding static variables shared between mapper and reducer classes.